src/components/ScriptureWorkspaceCard.js

```diff
@@ -115,7 +115,7 @@ export default function ScriptureWorkspaceCard({
             docSetId={docSetId}
             usfmText={data.usfmText}
             onSave={ (bookCode,usfmText) => setDoSave(usfmText) }
-            editable={id.endsWith(owner) ? true : false}
+            editable={data.id.endsWith(owner) ? true : false}
```

Comment:
I think you didn't intend these changes to be in this PR. They would be noticeable changes.